blackbox/docs/administration/system-information.txt

```diff
@@ -480,16 +499,33 @@ The table schema is as follows:
 | ``os['cgroup']['mem']['limit_bytes']``          | The max. amount of user memory in the cgroup.        | ``STRING``  |
 +-------------------------------------------------+------------------------------------------------------+-------------+
 
-.. note::
-
-    Cgroup metrics only work if the stats are available from ``/sys/fs/cgroup/cpu``
-    and ``/sys/fs/cgroup/cpuacct``.
-
 The cpu information values are cached for 1s. They might differ from the actual
 values at query time. Use the probe timestamp to get the time of collection.
 When analyzing the cpu usage over time, always use ``os['probe_timestamp']`` to
 calculate the time difference between 2 probes.
 
+.. _os_cgroup_limitations:
+
+Cgroup Limitations
+..................
+
+.. NOTE::
+
+    Cgroup metrics only work if the stats are available from
+    ``/sys/fs/cgroup/cpu`` and ``/sys/fs/cgroup/cpuacct``.
+
+.. _os_uptime_limitations:
+
+Uptime Limitations
+..................
+
+.. NOTE::
+
+    os['uptime'] required a system call when running CrateDB on Windows or
+    macOS, however, system calls are not permitted by default. If you require
+    this metric you need to allow system calls by setting ``bootstrap.seccomp``
```

Comment:
shouldn't we add this setting to the configuration section and link to there so that the user knows that if it's runtime or not, etc. ?

Reply:
it's an "undocumented" setting. I think we don't want users to fiddle with that.

Reply:
but we they don't know if it's runtime, or they have to put in the config file or... I would at least make it more specific here